### ISSUE TITLE
Peak flux greater than total flux?

### ISSUE THREAD

**Issue by cyriltasse** [lofar-astron/PyBDSF#42]
Simulating an image with point source, I was very surprised to see that the estimated peak flux could be greater than the total flux... Is that an expected behaviour?

![image](https://user-images.githubusercontent.com/6523227/33068108-8e35a286-ceb0-11e7-9f6c-f049f09f8f27.png)

**Comment by darafferty** [comment]
Yeah, it is allowed to fit Gaussians smaller than the restoring beam, so this can happen (especially with low S/N sources). You can freeze the size of the Gaussians to the beam with `fix_to_beam = True` if you know all sources are point sources.

**Comment by cyriltasse** [comment]
But isn't pybdsf fitting for `Gaussian(Total_flux_deconv,Sig_maj_deconv,Sig_min_deconv) x restoring_beam` where `x` is a convolution? In that case how is it even possible? Or maybe it's fitting for `Gaussian(Total_flux,Sig_maj,Sig_min)` and then given `restoring_beam` it's computing `Total_flux_deconv,Sig_maj_deconv,Sig_min_deconv,Peak_flux_deconv` thresholding `Sig_maj_deconv,Sig_min_deconv`. Cos I don't get consistent values between Total_flux/Peak_flux and `Sig_maj_deconv*Sig_min_deconv/(restoring_beam**2)`. I'm very confused...

**Comment by darafferty** [comment]
Yes, the second method is used: it fits the source then calculates the total flux as `peak*size[0]*size[1]/(bm_pix[0]*bm_pix[1])` where `bm_pix` is the restoring beam size and size is given by `Sig_maj` and `Sig_min` (i.e., not the deconvolved size).

**Comment by cyriltasse** [comment]
ok - sorry - I made a mistake above - so I meant `Sig_maj*Sig_min/(restoring_beam**2)` and by `Sig` one should  understand `FWHM`. I'll double check - but I wasn't finding `total=peak*size[0]*size[1]/(bm_pix[0]*bm_pix[1])`
